javascript-challenges.test.js

```diff
@@ -14,30 +14,122 @@
 
 // a) Create a test with expect statements using the variables provided.
 
-const secretCodeWord1 = "Lackadaisical"
+describe("codedMessage", () => {
+  it("returns a message that converts 'a' to 4, 'e' to 3, 'i' to 1, and 'o' to 0 ", () => {
+    expect(codedMessage(secretCodeWord1)).toEqual("L4ck4d41s1c4l");
+    expect(codedMessage(secretCodeWord3)).toEqual("3cc3ntr1c");
+  });
+});
+
+const secretCodeWord1 = "Lackadaisical";
 // Expected output: "L4ck4d41s1c4l"
-const secretCodeWord2 = "Gobbledygook"
+const secretCodeWord2 = "Gobbledygook";
 // Expected output: "G0bbl3dyg00k"
-const secretCodeWord3 = "Eccentric"
+const secretCodeWord3 = "Eccentric";
 // Expected output: "3cc3ntr1c"
 
 // b) Create the function that makes the test pass.
 
+const codedMessage = (string) => {
```

Comment:
Logic looks great!  Consider refactoring to a higher order function if you are looking for an additional challenge.  Solving the same problem multiple ways will make you an even better developer as you will be practicing with many different tools.